src/mon/MDSMonitor.cc

```diff
@@ -1954,8 +1972,13 @@ bool MDSMonitor::maybe_expand_cluster(std::shared_ptr<Filesystem> fs)
       break;
     }
 
-    dout(1) << "adding standby " << pending_fsmap.get_info_gid(newgid).addr
+    const auto &new_info = pending_fsmap.get_info_gid(newgid);
+    dout(1) << "adding standby " << new_info.addr
```

Comment:
"adding standby" is confusing to me; let's change the wording. It should say "promoting standby"?